tetris.py

```diff
@@ -74,14 +88,21 @@ def run_gui():
     sidebar = tkinter.Frame(root, bg=D_GREY)
     sidebar.grid(row=1, column=1, sticky="nsw")
 
-    new_game_button = tkinter.Button(sidebar, text="start")
-    new_game_button.grid(sticky="n")
+    #image source https://cooltext.com/
```

Comment:
I love this comment. It's as short as it could possibly be but it contains all the information needed. Beautiful!

Reply:
♥